packages/contentstack-import/src/lib/import/locales.js

```diff
@@ -48,11 +51,48 @@ module.exports = class ImportLanguages {
       self.langUidMapper = self.langUidMapper || {};
     }
 
-    return new Promise(function (resolve, reject) {
+    return new Promise(async function (resolve, reject) {
       if (self.languages === undefined || isEmpty(self.languages)) {
         addlogs(self.config, chalk.white('No Languages Found'), 'success');
         return resolve({ empty: true });
       }
+
+      let sourceMaterLangDetails = Object.values(self.sourceMasterLanguages);
+      if (self.masterLanguage["code"] === sourceMaterLangDetails[0]["code"]) {
+        self.masterLangDetails = await self.stackAPIClient.locale('en-us').fetch()
```

Comment:
@aman19K 'un-us' has been used as a fixed master locale code, it has to be fetch from the stack

Reply:
@shafeeqd959 fixed